frontend/src/metabase/visualizations/lib/settings/graph.js

```diff
@@ -146,7 +146,7 @@ export const GRAPH_DATA_SETTINGS = {
             ? t`Add series breakout`
             : null,
         columns: data.cols,
-        showColumnSetting: true,
+        showColumnSetting: [0],
```

Comment:
TBH, seeing this alone feels a bit weird. I wish we could add type to this, so it's easier to understand what this does. Maybe a comment could or giving the value a name would help with that?

Reply:
Yeah, I feel that. Especially when we don't have some of these things documented. I'll add a comment explaining the prop

Reply:
Maybe `showForIndices` or something like that would be more clear

Reply:
I'd really like to avoid changing the name of the prop, unless you're suggesting I add separate prop and resolve it in `ChartSettingFieldsPicker`

Reply:
A separate prop sounds better too because future engineers may struggle to understand `showColumnSetting` potentially be a boolean or an array

Reply:
I hate it when you're right haha. But I agree. I'll update this PR